mythx_cli/analyze/command.py

```diff
@@ -141,7 +170,7 @@ def analyze(
         create_group = analyze_config.get("create-group", False)
 
     mode = mode or analyze_config.get("mode") or "quick"
-    group_id = analyze_config.get("group-id") or group_id
+    group_id = analyze_config.get("group-id") or group_id or None
```

Comment:
why was this added now as part of the scribble integration?

Reply:
This is a simple fix on the side that makes the parameter definition more explicit. That way it's easier to see the state of `group_id` if neither a config file entry nor a CLI parameter are given. It's somewhat unrelated to scribble.